### ISSUE TITLE
LAST UPDATED FEATURE:  Make fixtures document showing sample JSON for the updates

### ISSUE THREAD

**Issue by cameron-barrett** [ucsb-cs156-s23/proj-courses-s23-7pm-1#15]
The data behind this table should be a MongoDB collection rather than a table in the SQL database; this is a kind of "metadata" for the MondoDB courses collection (see example), so it should be stored in an adjacent collection. We propose the name `updates`, and a JSON object with this structure:

```json
{
  "subject_area" : "CMPSC",
  "quarter_yyyyq": "20234",
  "last_update": "2023-05-13T10:22:35"
}
```

* In parallel with all other LAST UPDATED FEATURES, make a fixtures document in the frontend code showing sample JSON for the updates (see sample JSON above).  This can be a separate PR, or combined with PRs for the features below.
